### PR TITLE
Adding junk file paths to gitignore

### PR DESCRIPTION
**Pull request by gkiran292**

### CODE REVIEW

**Comment by gkiran292** [comment]
There are lot of eclipse-related hence I am adding gitignore to remove those files to make the FTP PR cleaner

**Comment by gkiran292** [comment]
Yes, we can specify the languages and platforms that need to be included. I used VSCode plugin to generate the .ignore file

**Comment by DImuthuUpe** [comment]
Can you add eclipse ignore content as well? And properly separate out ignore sections per ide

File: .gitignore [hunk 1]
```diff
@@ -1,34 +1,1208 @@
...
+*.db
+*.dbf
+*.mdb
+*.sqlite3
+
+### DBeaver ###
+# ide config file
+.dbeaver-data-sources*.xml
+
+### Eclipse ###
```
Comment (gkiran292) [inline .gitignore:104]: The autogenerator automatically comments about plaform too. I have included .gitignore of some python and other language files too as we would be building clients in the future.

**Comment by gkiran292** [comment]
> Can you add eclipse ignore content as well? And properly separate out ignore sections per ide

https://github.com/apache/airavata-mft/pull/10/files#r418706495

**Pull request merged**